cmd/start.go

```diff
@@ -35,30 +49,47 @@ func startNetwork(cmd *cobra.Command, args []string) error {
 		snapshotName = args[0]
 		startMsg = fmt.Sprintf("Starting previously deployed and stopped snapshot %s...", snapshotName)
 	} else {
-		snapshotName = defaultSnapshotName
+		snapshotName = constants.DefaultSnapshotName
 		startMsg = "Starting previously deployed and stopped snapshot"
 	}
 
 	ctx := binutils.GetAsyncContext()
 
 	ux.Logger.PrintToUser(startMsg)
-	_, err = cli.LoadSnapshot(ctx, snapshotName)
+	loadSnapshotOpts := []client.OpOption{
+		client.WithPluginDir(pluginDir),
+		client.WithExecPath(avalancheGoBinPath),
+	}
+	_, err = cli.LoadSnapshot(
+		ctx,
+		snapshotName,
+		loadSnapshotOpts...,
+	)
 	if err != nil {
-		return fmt.Errorf("failed to start network with the persisted snapshot: %s", err)
+		if !strings.Contains(err.Error(), "already bootstrapped") {
+			return fmt.Errorf("failed to start network with the persisted snapshot: %s", err)
+		}
+		ux.Logger.PrintToUser("Network has already been booted. Wait until healthy...")
+	} else {
+		ux.Logger.PrintToUser("Booting Network. Wait until healthy...")
 	}
 
 	// TODO: this should probably be extracted from the deployer and
 	// used as an independent helper
-	sd := subnet.NewLocalSubnetDeployer(log, baseDir)
-	endpoints, err := sd.WaitForHealthy(ctx, cli, healthCheckInterval)
+	_, _, endpoints, err := sd.WaitForHealthy(ctx, cli, healthCheckInterval)
 	if err != nil {
 		return fmt.Errorf("failed waiting for network to become healthy: %s", err)
 	}
 
 	fmt.Println()
-	ux.Logger.PrintToUser("Network ready to use. Local network node endpoints:")
-	for _, u := range endpoints {
-		ux.Logger.PrintToUser(u)
+	if len(endpoints) == 0 {
+		ux.Logger.PrintToUser("Network ready to use. No local network node endpoints")
```

Comment:
Not sure I follow. Why can there be 0 endpoints and how does this mean that the network is ready to use? What does it mean "No local network node endpoints"?

Reply:
Usually this and deploy prints `Local network node endpoints` before the blockchains list. As in this case there are no blockchains yet, only an empty network running, It seemed reasonable to me to print that. But we can:
1) change the message
2) avoid the situation of starting a network without blockchains (a little more work to have some state for the default snapshot contents but can be done)